client/packages/invoices/src/StockOut/utils.tsx

```diff
@@ -93,28 +93,43 @@ export const createDraftStockOutLineFromStockLine = ({
 export const createDraftStockOutLine = ({
   invoiceLine,
   stockLine,
-  adjustTotalNumberOfPacks = false,
-}: DraftStockOutLineSeeds): DraftStockOutLine => ({
-  isCreated: !invoiceLine,
-  isUpdated: false,
-  ...invoiceLine,
-  // When creating a draft outbound from an existing outbound line, add the available quantity
-  // to the number of packs. This is because the available quantity has been adjusted for outbound
-  // lines that have been saved.
-  // It's a similar story for totalNumber of packs if the invoice is in a picked state.
-  ...(stockLine
-    ? {
-        stockLine: {
-          ...stockLine,
-          availableNumberOfPacks:
-            stockLine.availableNumberOfPacks + invoiceLine.numberOfPacks,
-          totalNumberOfPacks: adjustTotalNumberOfPacks
-            ? stockLine.totalNumberOfPacks + invoiceLine.numberOfPacks
-            : stockLine.totalNumberOfPacks,
-        },
-      }
-    : {}),
-});
+  invoiceStatus,
+}: DraftStockOutLineSeeds): DraftStockOutLine => {
+  // When creating a draft stock out line from an invoice line we may need to adjust the available and total number of packs
+  // This is because, once an invoice line is created and it is in the allocated state, the available number of packs is reduced by the number of packs in the invoice line
+  // After it is in picked status, the total number of packs is also reduced by the number of packs in the invoice line
+  // Other statuses such as Shipped shouldn't show the stock line as available, so we don't need to adjust the available number of packs
+  // If the invoice is New, no adjustments are needed, as the stockLines shouldn't be updated yet
+
+  const adjustAvailableNumberOfPacks =
```

Comment:
I am pretty sure we adjustAvailableNumberOfPacks for all invoice statuses

Reply:
apart from new right? Then the lines don't actually exist. Although I think we don't call this code when the invoice is new... We call the draftFromStockLIne one instead...

Reply:
What do we want to do with Shipped for example I'm ok with always adjusting I'm just not sure if adjusting really makes sense...

Reply:
> apart from new right? Then the lines don't actually exist. Although I think we don't call this code when the invoice is new... We call the draftFromStockLIne one instead...

You can add and edit lines in new status.

>  I'm just not sure if adjusting really makes sense...

We've been following mSupply logic for this (even thought statuses are different, adjustment for available always happens)